src/gui_common/SpeciesPreview.cs

```diff
@@ -19,6 +19,8 @@ public Species? PreviewSpecies
 
             if (previewSpecies != null)
                 UpdatePreview();
+            else
```

Comment:
Our code style requires braces for all non-single line if statements, so please add braces here.